pandas/tests/io/test_common.py

```diff
@@ -169,6 +175,23 @@ def test_read_fspath_all(self, reader, module, path, datapath):
         mypath = CustomFSPath(path)
         result = reader(mypath)
         expected = reader(path)
+
+        if path.endswith('.pickle'):
```

Comment:
why was this change needed?

Reply:
There is no change. These six lines repeat at the bottom of `test_read_fspath_all_read_table` below, `diff` just does not realise that those six lines are the ones that "ought" to be highlighted instead of these six lines.